src/components/SubmissionModal/SubmissionModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { Link } from "react-router-dom";
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+import '../../styles/SubmissionModal.scss'
+import { borderRadius } from '@mui/system';
+
+const style = {
```

Comment:
Maybe we can make a page for these styles so we can export them and I can use those on the page views I am working on for consistency.